packages/search-token-lambda/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@coveo/search-token-lambda",
+  "version": "1.19.0",
+  "private": "true",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:coveo/cli.git",
+    "directory": "packages/search-token-lambda"
+  },
+  "scripts": {
+    "start:dev": "netlify functions:serve",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "npm:bump:template": "npm --no-git-tag-version version",
+    "npm:publish:template": "npm publish --access public --tag latest --registry http://localhost:4873"
+  },
+  "dependencies": {
+    "@coveord/platform-client": "^26.0.1",
+    "abortcontroller-polyfill": "^1.7.3",
+    "isomorphic-fetch": "^3.0.0"
+  },
+  "devDependencies": {
+    "@types/jest": "27.0.2",
+    "@typescript-eslint/eslint-plugin": "5.4.0",
+    "eslint": "8.2.0",
+    "gts": "3.1.0",
+    "jest": "27.3.1",
+    "netlify-cli": "^6.14.25",
+    "ts-jest": "27.0.7",
+    "typescript": "^4.5.2"
+  },
+  "publishConfig": {
+    "access": "public"
+  }
```

Comment:
Should we publish it already, or do it at the same time as `@coveo/create-atomic` ?

Reply:
Already!